lib/ansible/modules/apt.py

```diff
@@ -690,26 +691,36 @@ def parse_diff(output):
     return {'prepared': '\n'.join(diff[diff_start:diff_end])}
 
 
-def mark_installed_manually(m, packages):
+def mark_installed(m: AnsibleModule, packages: list[str], manual: bool) -> None:
+    """Mark packages as manually or automatically installed."""
     if not packages:
         return
 
+    if manual:
+        mark_state = "unmarkauto"
+        mark_msg = "manually"
+        mark_op = "manual"
+    else:
+        mark_state = "markauto"
```

Comment:
I just realized that we can remove `mark_state`, since it's been obsolete for over 14 years:

```suggestion
    if manual:
        mark_msg = "manually"
        mark_op = "manual"
    else:
```

Reply:
Didn't realize that.